exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `method` (optional): HTTP method, one of: `POST` or `PUT`. `POST` is used by default if unset.
```

Comment:
TBH i dont think we need to expose PUT.  I believe collection only supports POST, might be wrong would be worth asking the collection team.  But i don't like exposing something that a customer would never change.

Reply:
cc: @pmalek-sumo and https://github.com/influxdata/telegraf/tree/master/plugins/outputs/sumologic

Reply:
Thanks, will create a PR to correct it